src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug = '' } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'].map(column => (
```

Comment:
👍 but it would be better to create a `const` value for this array and then use this value here